insights/parsers/dnf_conf.py

```diff
@@ -0,0 +1,38 @@
+"""
+DnfConf - file ``/etc/dnf/dnf.conf``
+====================================
+
+This module provides parsing for the ``/etc/dnf/dnf.conf`` file.
+The ``DnfConf`` class parses the information in the file
+``/etc/dnf/dnf.conf``. See the ``YumConf`` class for more
+information on attributes and methods.
+
+Sample file content::
+
+    [main]
+    gpgcheck=1
+    installonly_limit=3
+    clean_requirements_on_remove=True
+    best=False
+    skip_if_unavailable=True
+
+Examples:
+    >>> 'main' in dconf
+    True
+    >>> 'rhel-7-server-rpms' in dconf
+    False
+    >>> dconf.has_option('main', 'gpgcheck')
+    True
+    >>> dconf.has_option('main', 'foo')
+    False
+"""
+
+from insights import parser
+from insights.specs import Specs
+from insights.parsers.yum_conf import YumConf
+
+
+@parser(Specs.dnf_conf)
```

Comment:
You probably should add a filter to this module to ensure that the input is minimally parsable.  Something like `add_filter(Specs.dnf_conf, '[')` should be sufficient.